VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.ts

```diff
@@ -65,10 +65,6 @@ export class JudgeWaitingRoomComponent extends WaitingRoomBaseDirective implemen
         return this.panelStates['Chat'];
     }
 
-    get areParticipantsVisible() {
-        return this.panelStates['Participants'];
-    }
```

Comment:
This was moved so that JOH users could also have access to this method for private consultations.